### PR TITLE
Cluster broadcast

### PR DESCRIPTION
**Pull request by marcinbudny**
This PR adds ability to broadcast events in a similar way the .NET version does. A new function on MemberList allows to pass event to all members of the cluster, that can be subscribed via EventStream.

### CODE REVIEW

File: cluster/member_list.go [hunk 1]
```diff
@@ -212,3 +213,13 @@ func groupMembersByKind(members []*Member) map[string][]*Member {
 	}
 	return groups
 }
+
+func (ml *MemberList) BroadcastEvent(message interface{}) {
```
Comment (marcinbudny) [inline cluster/member_list.go:217]: This function does not have the `includeSelf` parameter like the .NET counterpart. I don't think the .NET one is implemented correctly, as it compares the member id with actor system id, with former coming from cluster provider, and latter being a generated guid.

Comment (rogeralsing) [reply cluster/member_list.go:217]: >member id with actor system id

They are the same, the actor system id is the one registered into the cluster provider

File: cluster/member_list.go [hunk 2]
```diff
@@ -212,3 +213,13 @@ func groupMembersByKind(members []*Member) map[string][]*Member {
...
+	ml.mutex.RLock()
+	defer ml.mutex.RUnlock()
+
+	for _, member := range ml.members {
+		pid := actor.NewPID(member.Address(), "eventstream")
+		ml.cluster.ActorSystem.Root.Send(pid, message)
```
Comment (rogeralsing) [inline cluster/member_list.go:223]: 👍🏻

File: cluster/member_list.go (hunk 1 above)
Comment (marcinbudny) [reply cluster/member_list.go:217]: Correct, I did not follow this deep enough to see that e.g. Kubernetes provider sets pod label to actor system id.

**Pull request merged**